tools/helper/make_issue_transaction.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+import sys
+
+(gm, ca, da, index) = sys.argv
```

Comment:
add comment for variable meaning here plz
or rename them to more self-explainable names